### DISCUSSION TITLE
Variant is not showing over AR experience

### DISCUSSION THREAD

**Discussion by LuanCGale** [google/model-viewer#3330] [Q&A] [answered]
Hi @elalish or any one in the model viewer team,
I saw you mentioned in this ticket(https://github.com/google/model-viewer/discussions/3316) that the variant can be shown inside AR experience as well; however, I've tried to use the editor and download the scene with it and then try to run local server and test on my Android device; before going into AR, the variant all works nicely in Dom, but once go into AR, the variant disappeared and I can only see the default variant even if I switched from Dom. Is there anything I am missing? Thank you!

**Comment by elalish** [comment, marked as answer]
It won't work in Scene Viewer (since that's a separate app). Try WebXR AR mode instead. If that doesn't launch, check your console for errors; it requires HTTPS for instance. You can see an example of it working here: https://modelviewer.dev/examples/scenegraph/#variants

**Reply by LuanCGale** [reply]
Hi @elalish, 
Thanks, once I test it in HTTPS and it works, appreciate your reply!
